torch/onnx/verification.py

```diff
@@ -199,6 +199,17 @@ def _compare_ort_pytorch_outputs(
                         f"within acceptable range {acceptable_error_percentage}."
                     )
                     continue
+                else:
+                    if ort_out.is_quantized and pt_out.is_quantized:
```

Comment:
@jazzysoggy  ort_out is a bumpy array and doesn’t have the is_quantized attribute. I’d check it’s dtype and infer that it is a quantized tensor.

Reply:
Used dtype to infer quantization by checking whether the dtype is uint8 or int8